src/main/java/com/cdancy/bitbucket/rest/fallbacks/BitbucketFallbacks.java

```diff
@@ -135,11 +123,27 @@ public static MergeStatus createMergeStatusFromErrors(List<Error> errors) {
     /**
      * Parse list of Error's from output.
      *
-     * @param output json containing errors hash
+     * @param throwable
+     *            The error to report
      * @return List of Error's or empty list if none could be found
      */
-    public static List<Error> getErrors(String output) {
-        JsonElement element = parser.parse(output);
+    public static List<Error> getErrors(Throwable throwable) {
+        checkNotNull(throwable, "throwable");
+
+        JsonElement element;
+        try {
+            element = parser.parse(throwable.getMessage());
+        } catch (Exception e) {
+            // Happens when getMessage contains malformed JSON. Perhaps other
```

Comment:
Yes it will fail, and it did fail before. But the exception was lost. The stacktrace and message that says exactly what was actually wrong was lost.